fe/fe-core/src/main/java/com/starrocks/common/proc/CurrentQueryStatisticsProcDir.java

```diff
@@ -47,26 +48,26 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 /*
  * show proc "/current_queries"
  */
 public class CurrentQueryStatisticsProcDir implements ProcDirInterface {
     private static final Logger LOG = LogManager.getLogger(CurrentQueryStatisticsProcDir.class);
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
-                                                                    .add("QueryId")
-                                                                    .add("ConnectionId")
-                                                                    .add("Database")
-                                                                    .add("User")
-                                                                    .add("ScanBytes")
-                                                                    .add("ScanRows")
-                                                                    .add("MemoryUsage")
-                                                                    .add("DiskSpillSize")
-                                                                    .add("CPUTime")
-                                                                    .add("ExecTime")
-                                                                    .build();
-
-    private static final int EXEC_TIME_INDEX = 8;
+            .add("StartTime")
```

Comment:
new added fields are better to append to the end

Reply:
i prefer to maintain the semantic order instead of historical order.